tf/test.sh

```diff
@@ -15,17 +15,32 @@ echo "-> Creating the underlying infrastructure..."
 terraform init | tee tf.log
 terraform apply -auto-approve | tee -a tf.log
 
-AWS_REGION=$(terraform output -raw aws_region)
+# https://github.com/hashicorp/setup-terraform/issues/167#issuecomment-1090760365
+if [[ -z "${GITHUB_WORKFLOW}" ]]; then
```

Comment:
what do you think by using a binary alias? See below 
```suggestion
TERRAFORM_WRAPPER=terraform-bin
if [[ -z "${GITHUB_WORKFLOW}" ]]; then
  TERRAFORM_WRAPPER=terraform
fi
AWS_REGION=$(${TERRAFORM_WRAPPER} output -raw aws_region)
```

Reply:
https://github.com/elastic/apm-aws-lambda/pull/370/commits/f4e7c7c6a67de0665372e12dfde7c6817ec2e4fd